resources/README.md

```diff
@@ -10,13 +14,61 @@
 ## Developer Relations Surveys
 - [State of Developer Relations](https://www.stateofdeveloperrelations.com/) by Hoopy (DevRelCon)
 
-## Mindee assets
-- Ads image: 1920x1080 ([PNG](mindee/ad-1920x1080.png))
-- Ads video: 1920x1080 ([MP4](mindee/ad-1920x1080.mp4))
-- Logo ([SVG](mindee/logo.svg))
-- Logo Icon only ([SVG](mindee/logo-icon.svg))
+## Mindee Assets
 
-## Pictures
+### Ads
+- Image: 1920x1080 ([PNG](mindee/ad-1920x1080.png))
+- Video: 1920x1080 ([MP4](mindee/ad-1920x1080.mp4))
+
+### Logos (images)
+- [275x160 PNG](mindee/Logo-275x160.png)
+
+### Logos (vectors)
+- [Logo black](mindee/logo.svg)
+- [Icon only](mindee/logo-icon.svg)
+
+
+## Mindee Features Descriptions
+
+**Extract Key Information from any Type of Documents with ease**
```

Comment:
**Extract Key Information from any Type of Documents with ease**: Document processing and information extraction leveraging computer vision and machine learning through REST API.